src/Pyz/Zed/AntelopeDataImport/AntelopeDataImportConfig.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Pyz\Zed\AntelopeDataImport;
+
+use Spryker\Zed\DataImport\DataImportConfig;
+
+class AntelopeDataImportConfig extends DataImportConfig
+{
+    public const IMPORT_TYPE_ANTELOPE = 'antelope';
```

Comment:
Isn't CI complaining about a missing DockBlock here?

Reply:
nope, not complaining. Also the code sniff style did not complain. I added it manually